.github/actions/veracode/action.yml

```diff
@@ -4,20 +4,27 @@ inputs:
   srcclr-api-token:
     description: "Agent API Token"
     required: true
-  srcclr-install-default-options:
-    description: Default options for Source Clear scan
-    required: false
-    default: '-DskipTests -Dmaven.javadoc.skip=true'
-  srcclr-install-options:
-    description: Additional options for Source Clear scan
+  srcclr-project-ext:
+    description: "Veracode project name extension"
     required: false
 runs:
   using: "composite"
   steps:
+    - name: "Install srcclr CLI tool"
+      shell: bash
+      run: |
+        # Install srcclr CLI tool (Veracode) as per https://docs.veracode.com/r/t_sc_cli_agent
+        curl -sSL 'https://keyserver.ubuntu.com/pks/lookup?op=get&search=0xdf7dd7a50b746dd4' | sudo gpg --dearmor -o /usr/share/keyrings/veracode-sca-archive.gpg
+        echo 'deb [signed-by=/usr/share/keyrings/veracode-sca-archive.gpg] https://download.sourceclear.com/ubuntu stable/' | sudo tee /etc/apt/sources.list.d/veracode-sca.list
+        sudo apt-get update
+        sudo apt-get install srcclr
+    - name: "Add Veracode project name extension"
+      shell: bash
+      if: srcclr-project-ext != ''
+      run: |
+        echo "SRCCLR_SCM_URI=${{ github.server_url }}/${{ github.repository }}/${{ inputs.srcclr-project-ext }}" >> $GITHUB_ENV
```

Comment:
Adding such a suffix lets us use action for 
alfresco_enterprise_repo for branches 23.N, 7.4.N, 7.3.N and 7.2.N.
Without it we'd get results from all of them in the same project on Veracode side.
Adding "7.4.N" as srcclr-project-ext, will create 
alfresco_enterprise_repo_7.4.N project on veracode side, which can be conviniently linked to application version in Veracode. Just set a default branch for this project - release/7.4.N and link it to the ACS_EXT_MASTER_7_4 app.